app/lib/shared/handler_helpers.dart

```diff
@@ -67,10 +67,11 @@ shelf.Handler wrapHandler(
   }
   handler = _httpsWrapper(handler);
   handler = _logRequestWrapper(logger, handler);
+  handler =
+      _requestContextWrapper(handler); // need to run after session wrapper
```

Comment:
Maybe not that handlers wrap other handlers, so they are listed in reverse order...